packages/create-cosmos-app/src/git-cca-template.ts

```diff
@@ -166,7 +166,7 @@ export const createGitApp = (repo: string, version: string) => {
 
 
             // Construct the file path
-            const relativeFilePath = templateFile.split(join(folderName, template) + sep)[1];
+            const relativeFilePath = templateFile.split((join(folderName, template) + '/').replace(/\\/g,'/'))[1];
```

Comment:
Add this to the util and use it everywhere please

const posixPath = require('path').posix;

function toPosixPath(mixedPath) {
  return mixedPath.replace(/\\/g, posixPath.sep);
}